.chloggen/builder-configure-default-scheme.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: cmd/builder
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Allow setting `CollectorSettings.ResolverSettings.DefaultScheme` via the builder's `conf_resolver_settings.default_uri_scheme` configuration option
```

```suggestion
note: Allow setting `otelcol.CollectorSettings.ResolverSettings.DefaultScheme` via the builder's `conf_resolver_settings.default_uri_scheme` configuration option
```

Nit, but we should call out the package.